ru-ru/typescript-ru.html.markdown

```diff
@@ -0,0 +1,171 @@
+---
+language: TypeScript
+lang: ru-ru
+contributors:
```

Comment:
You should put your name into `translators`, not `contributors`. Contributors are those who wrote the original English article) Please correct.